miquiz_api.php

```diff
@@ -467,6 +467,10 @@ public static function scheduleTasks($miquiz)
             $gameModes[] = 'picked-fight';
         }
 
+        if (!empty($miquiz->game_mode_solo_fight)) {
+            $gameModes[] = 'solo-fight';
```

Comment:
In MI-Quiz we call this mode 'training', so here we should add 'training' instead of 'solo-fight'